auth/env_keychain.go

```diff
@@ -77,32 +87,38 @@ func ReadEnvVar(envVar string) (map[string]string, error) {
 	return authMap, nil
 }
 
-// BuildEnvVar creates the contents to use for authentication environment variable.
-//
-// Complementary to `ReadEnvVar`.
-func BuildEnvVar(keychain authn.Keychain, images ...string) (string, error) {
+func buildEnvMap(keychain authn.Keychain, images ...string) map[string]string {
```

Comment:
`buildEnvMap` -> `buildAuthMap`? `ResolveKeychain` uses this in ways that have nothing to do with the env var.